.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Infrastructure
+/infrastructure/                                      @agaudreault @leoluz
+/argocd/overlays/production/argocd-rbac-cm.yaml       @agaudreault @leoluz
+/external-dns/values.yaml                             @agaudreault @leoluz
```

Comment:
We should probably remove this for now until we decide how the ownership is going to be defined for the repo.

Reply:
👍 I commented it for now to keep track of infrastructure related files.